src/commonMain/libres/strings/time_units_cs.xml

```diff
@@ -5,11 +5,21 @@
         <item quantity="few">sekundy</item>
         <item quantity="other">sekund</item>
     </plurals>
+    <plurals name="seconds_past">
+        <item quantity="one">sekundou</item>
+        <item quantity="few">sekundami</item>
+        <item quantity="other">sekundami</item>
+    </plurals>
     <plurals name="minutes">
         <item quantity="one">minuta</item>
         <item quantity="few">minuty</item>
         <item quantity="other">minut</item>
     </plurals>
+    <plurals name="minutes_past">
+        <item quantity="one">minutou</item>
+        <item quantity="few">minutami</item>
+        <item quantity="other">minutami</item>
+    </plurals>
     <plurals name="hours">
```

Comment:
<plurals name="hours_past">
        <item quantity="one">hodinou</item>
        <item quantity="few">hodinami</item>
        <item quantity="other">hodinami</item>
    </plurals>
    <plurals name="weeks_past">
        <item quantity="one">týdnem</item>
        <item quantity="few">týdny</item>
        <item quantity="other">týdny</item>
    </plurals>
    <plurals name="months_past">
        <item quantity="one">měsícem</item>
        <item quantity="few">měsíci</item>
        <item quantity="other">měsíci</item>
    </plurals>
    <plurals name="years_past">
        <item quantity="one">rokem</item>
        <!-- "lety" is correct here too, but let's keep it synced with the "months" plurals, which also has "roky" for "few" type. -->
        <item quantity="few">roky</item>
        <item quantity="other">lety</item>
    </plurals>

Reply:
Please add these too ^^^. I probably didn't make my issue clear enough, so you've included only those mentioned in it. 🙂

Reply:
The issue was clear, no worries. I went to translation software and didn't notice a difference in these ones, but I must've overlooked it because now I do.

Thanks, I'll update the PR, stay tuned :)

Reply:
> "lety" is correct here too, but let's keep it synced with the "months" plurals, which also has "roky" for "few" type.

About this: I noticed this as well. Would it be nicer to align it all? So changing nominative `years.other` from let to roků/roky?

Reply:
Well, "let" is more common, so I'd personally choose the current implementation. Let me check some sources, so I can make a statement backed by something.

Reply:
Not necessarily needed, as I added the Czech localisation myself in the past (#7), so anything backed by a native speaker will be an improvement ;-)

Reply:
@jacobras Well, there's no strict rule for this, but Googling "roku vs. let" and going through the results gives me a confirmation we should keep this as it is.

Reply:
Okay! Would you mind taking a last look at the current state?

Reply:
@jacobras Looks good, thanks for the great work you're doing for the whole KMP community! If you'll ever need a CZ native again, let me know.

Reply:
Sure, thanks a lot for your help!